### ISSUE TITLE
[PREPROCESSING] Split train - val - test set

### ISSUE THREAD

**Issue by art-test-stack** [art-test-stack/vessel_position_forecast#14]
Do the split of the sets (train - val -test)

- Refacto the code name: TEST set should be only for the forecast window (data from ais_test.csv)
- In the split for the validation set, put the scaling factor as in the test set (meaning that the validation window will be on 5 days for each vessel, but does not need to be at the end of the training window)
